kmod/src/inode.c

```diff
@@ -749,7 +772,7 @@ struct inode *scoutfs_iget(struct super_block *sb, u64 ino, int lkf, int igf)
 	return inode;
 }
 
-static void store_inode(struct scoutfs_inode *cinode, struct inode *inode)
+static void store_inode(struct scoutfs_inode *cinode, struct inode *inode, int isize)
```

Comment:
isize is confusing, to fs people that reads as almost certainly something related to inode->i_size.   Maybe item_bytes?